pkg/archive/archive.go

```diff
@@ -525,7 +525,7 @@ loop:
 		if err != nil {
 			return err
 		}
-		if strings.HasPrefix(rel, "..") {
+		if strings.HasPrefix(rel, "../") {
```

Comment:
@tiborvass I guess not. Since it's unpack, it's daemon-side. We won't hit this yet on windows CI. but as @duglin said, it should use os.PathSeparator.